### PR TITLE
Updating checkpoint.ipynb by correcting a typo

### PR DESCRIPTION
**Pull request by chiron-bang**
Correction of a typo; from "vartiable's key" to "variable's key".

### CODE REVIEW

**Comment by chiron-bang** [comment]
@googlebot I signed it!

File: site/en/guide/checkpoint.ipynb [hunk 1]
```diff
@@ -520,7 +520,7 @@
       "source": [
         "### Manually inspecting checkpoints\n",
         "\n",
-        "`tf.train.load_checkpoint` returns a `CheckpointReader` that gives lower level access to the checkpoint contents. It contains mappings from each vartiable's key, to the shape and dtype for each variable in the checkpoint. A variable's key is its object path, like in the graphs displayed above.\n",
+        "`tf.train.load_checkpoint` returns a `CheckpointReader` that gives lower level access to the checkpoint contents. It contains mappings from each variable's key, to the shape and dtype for each variable in the checkpoint. A variable's key is its object path, like in the graphs displayed above.\n",
```
Comment (8bitmp3) [inline site/en/guide/checkpoint.ipynb:523]: Thanks for spotting this!

**Comment by 8bitmp3** [comment]
LGTM

**Review by 8bitmp3 — APPROVED** [review]
👍

**Pull request merged**